pkgs/development/python-modules/stanza/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib,
+buildPythonPackage,
+python3Packages,
+pythonOlder }:
+
+buildPythonPackage rec {
+  pname = "stanza";
+  version = "1.3.0";
+
+  src = python3Packages.fetchPypi {
```

Comment:
```suggestion
  src = fetchPypi {
```

Reply:
Changed to `fetchFromGitHub` without `python3Packages`.